### PR TITLE
refactor readme and remove some duplicated contents

### PR DESCRIPTION
**Pull request by njzjz**
Note: some documentation links will be available once the devel branch is merged into the master branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,1198 +36,30 @@ To test if the installation is successful, you may execute
...
-     &CELL
-       ABC LEFT FOR DPGEN
-     &END CELL
-     &COORD
-     @include coord.xyz
-     &END COORD
-   &END SUBSYS
-&END FORCE_EVAL
-```
+* [`dpgen run`](https://docs.deepmodeling.com/projects/dpgen/en/latest/run/): Main process of Deep Generator.
```
Comment (wanghan-iapcm) [inline README.md:43]: The link is broken

File: README.md [hunk 2]
```diff
@@ -99,1198 +36,30 @@ To test if the installation is successful, you may execute
...
 
-A property is tested in three steps: `make`, `run` and `post`. `make` prepares all computational tasks that are needed to calculate the property. For example to calculate EOS, `make` prepares a series of tasks, each of which has a scaled configuration with certain volume, and all necessary input files necessary for starting a VASP, ABACUS, or LAMMPS calculations. `run` sends all the computational tasks to remote computational resources defined in a machine configuration file like `machine.json`, and automatically collects the results when remote calculations finish. `post` calculates the desired property from the collected results.
+## Tutorials and examples
 
-### Relaxation
+* [Tutorials](https://tutorials.deepmodeling.com/en/latest/Tutorials/DP-GEN/): basic tutorials for DP-GEN.
+* [Examples](examples): input files in [JSON](https://docs.python.org/3/library/json.html) format.
+* [Publications](https://deepmodeling.com/blog/papers/dpgen/): Published research articles using DP-GEN.
+* [User guide](https://docs.deepmodeling.com/projects/dpgen/en/latest/user-guide/): frequently asked questions listed in troubleshooting.
```
Comment (wanghan-iapcm) [inline README.md:58]: this one is broken. I didn't check every link, could you please do so?

Comment (njzjz) [reply README.md:58]: These links will be available once the devel branch is merged to the master branch. At present the link can be visited by replacing `latest` with `devel`.

**Review by wanghan-iapcm — APPROVED** [review]

**Review by HuangJiameng — CHANGES_REQUESTED** [review]
dpgen package on conda-forge is version 0.9.2. Shall we change the link to https://anaconda.org/deepmodeling/dpgen, which is the latest version?

File: README.md [hunk 3]
```diff
@@ -1,84 +1,21 @@
...
-      * [License](#license)
-
-
-## About DP-GEN
+# DP-GEN: A concurrent learning platform for the generation of reliable deep learning based potential energy models
 
 [![GitHub release](https://img.shields.io/github/release/deepmodeling/dpgen.svg?maxAge=86400)](https://github.com/deepmodeling/dpgen/releases/)
 [![doi:10.1016/j.cpc.2020.107206](https://img.shields.io/badge/DOI-10.1016%2Fj.cpc.2020.107206-blue)](https://doi.org/10.1016/j.cpc.2020.107206)
 ![Citations](https://citations.njzjz.win/10.1016/j.cpc.2020.107206)
 [![conda install](https://img.shields.io/conda/dn/conda-forge/dpgen?label=conda%20install)](https://anaconda.org/conda-forge/dpgen)
```
[the remark this answers is not in the document]
Comment (njzjz) [reply README.md:6]: I will resolve the conda-forge issue this month.

[the remark this answers is not in the document]
Comment (njzjz) [reply README.md:6]: Resolved.

**Review by HuangJiameng — APPROVED** [review]

**Pull request merged**